auth0/src/main/java/com/auth0/android/provider/OAuthManager.java

```diff
@@ -320,6 +327,14 @@ private void addPKCEParameters(Map<String, String> parameters, String redirectUr
         }
     }
 
+    private void addPKCEHeaders(@NonNull Map<String, String> httpHeaders) {
```

Comment:
I'd merge this method into `addPKCEParameters`. Please, make the `createPKCE()` method take the headers in addition to the redirectUri. That way they are passed in the construction of the PKCE instance. That would as well remove the setter (pkce.setHeaders)